internal/api/contract/GetOrgResponse.go

```diff
@@ -0,0 +1,28 @@
+package contract
```

Comment:
Question: Does it make sense to generate this contract rather than manually maintaining it?

Reply:
We will postpone this topic a bit until we have a solution we can apply consistently.

Reply:
💯, but this sounds like another initiative which shouldn't block this PR